### ISSUE TITLE
NVDA is not announcing text when using inputgesture.send()

### ISSUE THREAD

**Issue by DalisonCPU** [nvaccess/nvda#17202]
### Steps to reproduce:
Create a script that detects the right or left arrow in a text.
Intercept the key and use the send method of the gesture class
### Actual behavior:

NVDA sends the requested key with the send method, but the text is not being read. For example, if there is a key on the left and I use the send method on the key to the left, it does not announce the positioned character.
### Expected behavior:

NVDA should announce the positioned character.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2024.3.1
#### Windows version:
Windows 11
#### Name and version of other software in use when reproducing the issue:
notepad
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
not
#### If NVDA add-ons are disabled, is your problem still occurring?
Needs an addon to test the bug
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by seanbudd** [comment]
This is intended behaviour. Closing as invalid. If you need add-on development support please send an email to [nvda-devel](https://groups.io/g/nvda-devel/topics) or create a [GitHub discussion](https://github.com/nvaccess/nvda/discussions)
